snapshot.sh

```diff
@@ -31,9 +32,15 @@ while true; do
         aws_args="--endpoint-url ${S3_HOST}"
         s3_uri_base="s3://${SNAPSHOT_PATH}"
         timestamp=$(date +"%Y-%m-%dT%H:%M:%S")
-        s3_uri="${s3_uri_base}/${SNAPSHOT_PREFIX}_${timestamp}.tar.gz"
+        s3_uri="${s3_uri_base}/${SNAPSHOT_PREFIX}_${timestamp}.${SNAPSHOT_FORMAT}"
 
-        tar c -C $SNAPSHOT_DIR . | gzip | aws $aws_args s3 cp - "$s3_uri" --expected-size $SNAPSHOT_SIZE
+        SNAPSHOT_SIZE=$(du -sb $SNAPSHOT_DIR | cut -f1)
```

Comment:
Maybe update the default value for `SNAPSHOT_SIZE` to `0` to enable auto calculation...
Or simply remove it from doc (Unless there is a meaning of manually inputting it when it can be calculated)

Reply:
Good point. fixed.